raiden-dapp/src/store/index.ts

```diff
@@ -54,6 +54,7 @@ const _defaultState: RootState = {
   config: {},
   userDepositTokenAddress: '',
   disclaimerAccepted: false,
+  latestStateBackupReminder: 0,
```

Comment:
Background: I remember that there was a "discussion" what representation/type to use here.
Unfortunately the name of the variable does not give enough information to explain itself. Is this the date in milliseconds when the last reminder was shown or..? Please check if you could help a stupid reader like me with a better name if possible. :wink: 

PS: I think the signature of this mutation (`stateBackupReminder(state: RootState, newReminderDate: number)`) shows perfectly the mismatch. Btw I think active names for mutations are preferrable. Also we did not in the past we should attempt to improve. Something like `update...`, `set...` or something else with a verb. `stateBackupReminder` what?!